templates/events/add-new-event.html

```diff
@@ -805,6 +810,30 @@ <h5>Add Sub-Event </h5>
   overflow-y: visible;
 }
 
+  .mytooltip {
+  position: relative;
+  display: inline-block;
+  border-bottom: 1px dotted black;
+}
+
+.mytooltip .mytooltiptext {
+  visibility: hidden;
+  width: 1200px;
+  background-color: black;
+  color: #fff;
```

Comment:
```suggestion
  color: #343a40;
```
Suggesting a color change.